packages/databricks-vscode/package.json

```diff
@@ -298,6 +310,12 @@
                     "visibility": "visible",
                     "when": "databricks.context.bundle.isTargetSet"
                 },
+                {
+                    "id": "dabsVariableView",
+                    "name": "Bundle Variable View",
+                    "visibility": "visible",
+                    "when": "databricks.context.bundle.isTargetSet"
```

Comment:
Empty variable view is confusing right now, especially since it lets you "set bundle variables", but nothing will actually show up untill you define the actual variables in the `databricks.yml`. Let's hide it when there's not variables?